install.sh

```diff
@@ -66,18 +72,20 @@ msg "Currently, you can install one of 'makedeb', 'makedeb-beta', or"
 msg "'makedeb-alpha'."
 msg "If you're not sure which one to choose, pick 'makedeb'."
 
-while true; do
-    read -p "$(question "Which release would you like? ")" response
+if ! is_valid_makedeb_pkg "$MAKEDEB_PKG"; then
```

Comment:
If a user passes in `MAKEDEB_PKG=asdf`, this will still go through the loop, but if the variable is set I think it'd just be better to exit immediately and print an error. How you had it [here](https://github.com/makedeb/makedeb/blob/6bbafcb81ab48007e3b6b6a2e4fb20af66238613/install.sh#L66-L79) was good, just adding an `elif` statement and then checking that the package is there should be fine